yarn-project/p2p/src/services/discv5/discv5_service.test.ts

```diff
@@ -348,8 +348,9 @@ describe('Discv5Service', () => {
     const peerId = await createSecp256k1PeerId();
     const config: P2PConfig = {
       ...getP2PDefaultConfig(),
-      ...baseConfig,
+      ...emptyChainConfig,
```

Comment:
This is the culprit.

...baseConfig was passed to the bootstrap node on line 59. So it was being mutated in between tests.

Reply:
Fatality.